DOC Update documentation for vote_for_n

ethanmad opened the pull request:
The docstrings were misleading about the following variables:
- compute_winner returns winners, a list of integers, where each integer
  represents the index of a candidate who won the election.
- total_num_votes is a list of the total number of votes, not an
  integer.

I also updated the HTML documentation, which was not updated to the
latest docstrings anyway. The PDF documentation is also outdated,
but I do not know how to update it. If you could explain, I will
gladly take care of it.

While I am here, thank you for the excellent tool.

ethanmad committed:
DOC Update documentation for vote_for_n

The docstrings were misleading about the following variables:
- compute_winner returns winners, a list of integers, where each integer
  represents the index of a candidate who won the election.
- total_num_votes is a list of per-county total number of votes, not
  an integer.

The sizes of these variables are also now clearly spcecified.

Signed-off-by: Ethan Madison <ethan@ethanmad.com>

mayuri95 commented:
Hi!

Thank you so much for taking a look at this - you're absolutely correct on the misleading docstrings. 

Could I request that in the docstrings that you update, you also put the length of the list - i.e. winners is a list of length k, where k is specified by the vote_for_n parameter, which defaults to 1?

On a minor note, there are also some small grammatical issues in the changes - for instance, total_num_votes is a list of _integers_, rather than "an integer".

Generating the updated PDF just requires saving the HTML page by doing "Print to PDF". It's just for convenience sake. Thank you again for taking a look at this - I'll merge the pull request, if you could please address my comments.

ethanmad commented:
Hi @mayuri95,

907502f addresses those issues as well as the originals.

It turned out I had misunderstood what `total_num_votes` is; in some places, it's an integer representing the total number of votes in a particular county; while in others, it is a list comprising those values for all counties. I don't use the multi-county functionality (yet?), so I didn't realize that it had per-county values.

mayuri95 commented:
thanks, @ethanmad!
